Report/sections/data_technology/completezza.tex

```diff
@@ -147,7 +147,12 @@ \subsubsection*{Weather}
 
 \subsubsection*{Stations}
 \addcontentsline{toc}{subsubsection}{Stations}
-Il dataset \textsc{Stations} presenta una completezza del 82,67\%. Esso risulta essere il meno completo tra i dataset presentati. In particolare quattro attributi su 15 presentano dei valori mancanti.\\
+Il dataset \textsc{Stations} presenta una completezza del 82,67\%. Esso risulta 
+essere il meno completo tra i dataset presentati. In particolare quattro 
```

Comment:
mi rendo conto ora che scriviamo quattro in lettere e quindici in numeri, forse dovremmo rimanenre consistenti

Reply:
Beh se i numeri sono piccoli ok, ma un certo punto devi anche iniziare a scriverli in numeri.